content/rs/administering/security/account-management.md

```diff
@@ -5,62 +5,163 @@ weight: $weight
+    <td align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+    <td align="center"><img src="../../../../images/icon_logo/icon_check_black.png#no-click" alt="V"></td>
+  </tr>
+</table>
+
+## Adding a user
+
+To add a user to the cluster:
+
+1. Go to: **settings** > **team**
+1. Click ![Add](/images/rs/icon_add.png#no-click "Add").
+1. Enter the name, email and password of the new user and select the role to assign to the user.
+1. Select the type of user:
+    - internal - Authenticates with RS
+    - external - Authenticates with an external LDAP server
+
+      To have a user authenticate with LDAP, you must first [enable LDAP integration]({{< relref
+      "/rs/administering/security/ldap-integration.md" >}}).
+
+1. For the email alerts, click **Edit** and select the alerts that the user receives.
+    You can select:
+    - Receive alerts for databases - The alerts that are enabled for the selected databases are sent to
+      the user. You can either select all databases, or you can select **Customize** and select the
+      individual databases to send alerts for.
+    - Receive cluster alerts - The alerts that are enabled for the cluster are send to the user.
+
+    {{% expand "Show me how..." %}}![Select email alerts](/images/rs/add-user-email-alerts.gif "Select email alerts"){{% /expand %}}
+
+    Then, click **Save**.
+1. Click ![Save](/images/rv/icon_save.png#no-click "Save").
+
+To edit the name, password, role or email alerts of a user, hover over the user and click ![Edit]
+(/images/rv/icon_edit.png#no-click "Edit"). To change a user from internal to external, you must
+delete the user and re-add it.
 
 ## Creating users for use with LDAP authentication
 
 To have a user authenticate with LDAP, you must have [LDAP integration
-enabled]({{< relref "/rs/administering/security/ldap-integration.md" >}}),
-and then create a new user via the REST API call like this:
+enabled]({{< relref "/rs/administering/security/ldap-integration.md" >}}). 
+Then, create a new **external** user in the web UI.
+
+You can also create an external with the REST API with this syntax:
 
 ```src
-curl -k -L -v -u ":" --location-trusted -H "Content-Type: application/json" -X POST http://:8080/v1/users -d "{\"auth_method\": \"external\", \"name\": \"\", \"role\": \"\"}"
+curl -k -L -v -u ":" --location-trusted -H "Content-Type: application/json" -X POST http://<RS_server_address>:8080/v1/users -d "{"auth_method": "external", "name": "<username>", "role": "<user_role>"}"
 ```
 
-For the user-role, put in one of the following roles:
+For the user role, enter either:
```

Comment:
1) I think this section should be part of the "Adding a user" section and not here
2) Please avoid using the "dev" representation of the roles. We'd better use the UI names: Admin, DB Viewer etc.